Blank identifier '_' causes panic

```go
package main

import (
	"bytes"
	"encoding/binary"
	"fmt"
	"github.com/nokute78/go-bit/pkg/bit/v2"
	"io"
)

type BitMap struct {
	Bit0     bit.Bit
	Bit1     bit.Bit
	Bit2     bit.Bit
	Bit3     bit.Bit
	Reserved [4]bit.Bit `bit:"skip"`
	_        [8]bit.Bit
}

func ReadBitMap(r io.Reader, b binary.ByteOrder) (*BitMap, error) {
	ret := &BitMap{}
	if err := bit.Read(r, b, ret); err != nil {
		return nil, err
	}
	return ret, nil
}

func main() {
	buf := bytes.NewBuffer([]byte{0xf5, 0xff})
	bm, err := ReadBitMap(buf, binary.LittleEndian)
	fmt.Printf("bm=%+v err=%s\n", bm, err)
}
```

```
$ go run ~/example.go 
panic: reflect.Value.Interface: cannot return value obtained from unexported field or method

goroutine 1 [running]:
reflect.valueInterface(0x4b7b80, 0xc00001a128, 0x1b1, 0x575b01, 0x7f1355397108, 0x0)
	/usr/local/go/src/reflect/value.go:1026 +0x169
reflect.Value.Interface(...)
	/usr/local/go/src/reflect/value.go:1015
github.com/goccy/go-reflect.value_Interface(...)
	/home/taka/go/pkg/mod/github.com/goccy/go-reflect@v1.1.0/value.go:144
github.com/goccy/go-reflect.Value.Interface(...)
	/home/taka/go/pkg/mod/github.com/goccy/go-reflect@v1.1.0/reflect.go:877
github.com/nokute78/go-bit/pkg/bit/v2.fillData(0xc00001a158, 0x2, 0x2, 0x4f3500, 0x5a4c88, 0x4b7b80, 0xc00001a128, 0x1b1, 0xc000066e48, 0x8, ...)
	/home/taka/go/src/github.com/nokute78/go-bit/pkg/bit/v2/decode.go:118 +0x85
github.com/nokute78/go-bit/pkg/bit/v2.fillData(0xc00001a158, 0x2, 0x2, 0x4f3500, 0x5a4c88, 0x4c6220, 0xc00001a120, 0x199, 0xc000066e48, 0x10, ...)
	/home/taka/go/src/github.com/nokute78/go-bit/pkg/bit/v2/decode.go:238 +0x698
github.com/nokute78/go-bit/pkg/bit/v2.Read(0x4f2800, 0xc00005c180, 0x4f3500, 0x5a4c88, 0x4b1fc0, 0xc00001a120, 0x4c1100, 0xc000010201)
	/home/taka/go/src/github.com/nokute78/go-bit/pkg/bit/v2/decode.go:283 +0x249
main.ReadBitMap(0x4f2800, 0xc00005c180, 0x4f3500, 0x5a4c88, 0x4420ca, 0x5604e0, 0xc000030778)
	/home/taka/example.go:22 +0x78
main.main()
	/home/taka/example.go:30 +0xa7
exit status 2
```

--- a/pkg/bit/v2/decode.go
+++ b/pkg/bit/v2/decode.go
@@ -18,6 +18,7 @@ package bit
 
 import (
 	"encoding/binary"
+	"errors"
 	"fmt"
 	"github.com/goccy/go-reflect"
 	"io"
@@ -27,6 +28,8 @@ import (
 const (
 	tagKeyName = "bit"
 )
+
+var errCannotInterface = errors.New("CanInterface returns false")
 
 // tagConfig represents StructTag.
 //   "-"   : ignore the field
@@ -115,6 +118,16 @@ func fillData(b []byte, order binary.ByteOrder, v reflect.Value, o *Offset) erro
 	var err error
 	var val reflect.Value
 
+	if !v.CanInterface() {
+		// skip unexported field
+		var size int
+		sizeOfValueInBits(&size, v, false)
+		*o, err = o.AddOffset(Offset{Bit: uint64(size)})
+		if err != nil {
+			return err
+		}
+		return errCannotInterface
+	}
 	d := v.Interface()
 
 	switch d.(type) {
@@ -203,7 +216,7 @@ func fillData(b []byte, order binary.ByteOrder, v reflect.Value, o *Offset) erro
 				} else {
 					for i := 0; i < v.Len(); i++ {
 						err := fillData(b, order, v.Index(i), o)
-						if err != nil {
+						if err != nil && err != errCannotInterface {
 							return err
 						}
 					}
@@ -229,14 +242,14 @@ func fillData(b []byte, order binary.ByteOrder, v reflect.Value, o *Offset) erro
 						continue
 					} else if cnf.endian != nil {
 						err := fillData(b, cnf.endian, v.Field(i), o)
-						if err != nil {
+						if err != nil && err != errCannotInterface {
 							return err
 						}
 						continue
 					}
 				}
 				err := fillData(b, order, v.Field(i), o)
-				if err != nil {
+				if err != nil && err != errCannotInterface {
 					return err
 				}
 			}
@@ -281,7 +294,7 @@ func Read(r io.Reader, order binary.ByteOrder, data interface{}) error {
 			return fmt.Errorf("bit.Read:short read, expect=%d byte, read=%d byte", byteSize, n)
 		}
 		err = fillData(barr, order, reflect.Indirect(v), &Offset{})
-		if err != io.EOF {
+		if err != io.EOF && err != errCannotInterface {
 			return err
 		}
 	default:

--- a/pkg/bit/v2/decode_test.go
+++ b/pkg/bit/v2/decode_test.go
@@ -339,3 +339,23 @@ func BenchmarkReadStruct(b *testing.B) {
 		}
 	}
 }
+
+func TestUnexportedField(t *testing.T) {
+	type S struct {
+		A [8]bit.Bit
+		_ [4]bit.Bit
+		B [4]bit.Bit
+	}
+
+	var s S
+	buf := bytes.NewBuffer([]byte{0xaa, 0xfb})
+	if err := bit.Read(buf, binary.LittleEndian, &s); err != nil {
+		t.Errorf("err=%s", err)
+	}
+	for i := 0; i < len(s.B); i++ {
+		if !s.B[i] {
+			t.Fatalf("val error: given=%v", s.B)
+
+		}
+	}
+}

